### PR TITLE
[thread-cert] refactor case 5.6.6 and 5.6.9 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_5_6_09_NetworkDataForwarding.py [hunk 1]
```diff
@@ -96,19 +104,21 @@ def test(self):
...
 
         # Set lowpan context of sniffer
         self.simulator.set_lowpan_context(1, '2001:2:0:1::/64')
 
         self.simulator.go(10)
 
-        self.nodes[ROUTER2].add_prefix('2001:2:0:1::/64', 'paros', 'low')
-        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', 'high')
+        self.nodes[ROUTER2].add_prefix('2001:2:0:1::/64', 'aos', 'med')
+        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', 's', 'high')
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_5_6_09_NetworkDataForwarding.py:119]:
```suggestion
        self.nodes[ROUTER2].add_route('2001:2:0:2::/64', stable=True, prf='high')
```
should use `True/False` for `stable` argument?

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 
with a nit

File: tests/scripts/thread-cert/Cert_5_6_09_NetworkDataForwarding.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_6_09_NetworkDataForwarding.py]: added

File: tests/scripts/thread-cert/Cert_5_6_06_NetworkDataExpiration.py [hunk 2]
```diff
@@ -154,6 +141,89 @@ def test(self):
...
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        SED = pv.vars['SED']
+        _lpkts = pkts.filter_wpan_src64(LEADER)
+
+        # Step 1: Ensure the topology is formed correctly
+        _lpkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).filter_wpan_dst64(SED).must_next()
+
+        # Step 4: The DUT Automatically sends a CoAP Response frame to Router_1
+        _lpkts.copy().filter_coap_ack(SVR_DATA_URI).must_next()
```
Comment (jwhui) [inline tests/scripts/thread-cert/Cert_5_6_06_NetworkDataExpiration.py:160]: Validate destination as Router_1?

Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_5_6_06_NetworkDataExpiration.py]: added

**Review by jwhui — APPROVED** [review]

**Pull request merged**